specs/agents/tracing-spans.md

```diff
@@ -95,6 +95,9 @@ spans that are slower than a specified duration, using the config variable
 `span_stack_trace_min_duration`. (Previously
 `span_frames_min_duration`.)
 
+Agents based on OpenTelemetry should capture this in a span attribute: `elastic.span.stack_trace`.
+Stack trace should be stored in plain text using the usual text format of the platform.
```

Comment:
Maybe link to https://opentelemetry.io/docs/specs/semconv/exceptions/exceptions-spans/#stacktrace-representation?